Makefile

```
@@ -14,11 +14,12 @@ info:
 .restart-cqf-ruler:
 	docker stop cqf-ruler || true && docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler:develop
```

```suggestion
	-docker stop cqf-ruler
	docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler:develop
```

Adding a `-` to the beginning of a line in the makefile will prevent make from halting upon error and I think is a slightly cleaner way of doing this